invenio_rdm_migrator/streams/records/load.py

```diff
@@ -14,7 +14,7 @@
 class RDMRecordCopyLoad(PostgreSQLCopyLoad):  # TODO: abstract SQL from PostgreSQL?
     """PostgreSQL COPY load."""
 
-    def __init__(self, db_uri, tmp_dir):
+    def __init__(self, communities_cache, db_uri, tmp_dir):
```

Comment:
minor: minimize breaking the signature (though is not a kwargs so anyway breaks, so is just me being picky)
```suggestion
    def __init__(self, db_uri, tmp_dir, communities_cache):
```

Reply:
I think I did it because the streams.yaml when I dump the communities_cache it adds the key first because I guess of the alphabetical order...